internal/controllers/httpsedge_controller.go

```diff
@@ -871,10 +940,11 @@ func (u *edgeRouteModuleUpdater) setEdgeRouteWebhookVerification(ctx context.Con
 	}
 
 	if reflect.DeepEqual(&module, route.WebhookVerification) {
-		u.log.Info("Webhook Verification matches desired state, skipping update")
+		log.V(1).Info("Webhook Verification matches desired state, skipping update")
```

Comment:
Nit: worth differentiating the DeepEquals messages from the nil?

Reply:
Good call. I can add those in where appropriate.

Reply:
Should be fixed in d5d7d57. Let me know what you think